inf-clojure.el

```diff
@@ -367,6 +367,7 @@ This should usually be a combination of `inf-clojure-prompt' and
   "Controls whether to display the REPL buffer in the current window or not."
   :type '(choice (const :tag "same" t)
                  (const :tag "different" nil))
+  :safe (lambda (s) (or (string-equal "same" s) (string-equal "different" s)))
```

Comment:
The actual values are `t` and `nil`, not "same" and "different". A guess this should just be a `boolean?`.